xcube_geodb_openeo/server/app/flask.py

```diff
@@ -38,57 +38,61 @@
     )
 )
 
+well_known = flask.Blueprint('well_known', __name__)
 api = flask.Blueprint('api', __name__, url_prefix=API_URL_PREFIX)
 
 ctx = AppContext(app.logger)
 
 
+@well_known.route('/.well-known/openeo')
+def get_well_known():
+    return capabilities.get_well_known(ctx.config)
+
+
 @api.route('/')
 def get_info():
-    return {
-        'name': __name__,
-        'version': __version__
-    }
+    return capabilities.get_root(
+        ctx.config, ctx.for_request(f'{flask.request.root_url}'
+                                    f'{api.url_prefix}')
+    )
```

Comment:
```suggestion
    return capabilities.get_root(
        ctx.config, ctx.for_request(f'{flask.request.root_url}'
                                    f'{api.url_prefix}'))
```

See comment on alternative indent style below.